tools/make_boot_descriptor.py

```diff
@@ -186,9 +186,9 @@ def length(self):
             self._contents.seek(0, os.SEEK_END)
             self._length = self._contents.tell()
             if self._padding:
-                fill = self._padding - (self._length % self._padding)
-                self._length += fill
-                self._padding = fill
+                mod = self._length % self._padding
+                self._padding = self._padding - mod if mod else 0
```

Comment:
I'm not sure I follow what is the purpose of `mod if mod else 0`. Couldn't this be replaced with just `mod`?

Reply:
The line is interpreted as `self._padding = (self._padding - mod) if mod else 0`.

Reply:
Ah. Confusing. Thanks though!